pyiron/atomistics/structure/atoms.py

```diff
@@ -21,7 +21,8 @@
     ChemicalElement
 )
 from pyiron_base import Settings
-from pyiron_base.generic.util import deprecate
+from pyiron_base.generic.util import deprecate, deprecate_soon
```

Comment:
To keep the interface clean I would suggest to import directly from `pyiron_base`: 
```
from pyiron_base import deprecate, deprecate_soon
```
As implemented in https://github.com/pyiron/pyiron_base/pull/122